job/job.go

```diff
@@ -90,47 +89,37 @@ func HasActiveWorkflow(ctx context.Context, hwID packet.HardwareID) (bool, error
 }
 
 // CreateFromDHCP looks up hardware using the MAC from cacher to create a job
```

Comment:
should document that the returned `ctx` is likely to be modified (even in error case) and should thus overtake the one used as the arg.